src/internal/operations/pathtransformer/restore_path_transformer_test.go

```diff
@@ -0,0 +1,293 @@
+package pathtransformer_test
+
+import (
+	"testing"
+
+	"github.com/alcionai/clues"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/alcionai/corso/src/internal/operations/pathtransformer"
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/internal/version"
+	"github.com/alcionai/corso/src/pkg/backup/details"
+	"github.com/alcionai/corso/src/pkg/backup/details/testdata"
+	"github.com/alcionai/corso/src/pkg/fault"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+type RestorePathTransformerUnitSuite struct {
+	tester.Suite
+}
+
+func TestRestorePathTransformerUnitSuite(t *testing.T) {
+	suite.Run(t, &RestorePathTransformerUnitSuite{Suite: tester.NewUnitSuite(t)})
+}
+
+func (suite *RestorePathTransformerUnitSuite) TestGetPaths() {
```

Comment:
could this get some tests for sharepoint pages and lists, to ensure they'll get handled properly here if/when we turn on their support?

Reply:
I'm honestly not sure what those paths should look like right now so I'm unsure what data to use for them. If you can provide some data I can certainly update the tests

Reply:
hmmm it may be best to hold off on other SharePoint stuff until we have a better idea of what it will look like. I will add a check in the code for SharePoint categories and return an error if it's not libraries. I'll also write a test case to check for that failure. That should hopefully keep us from missing it when we do add support